terraform/20-app/cloud-front.front-end.tf

```diff
@@ -81,6 +81,21 @@ module "cloudfront_front_end" {
       viewer_protocol_policy     = "redirect-to-https"
       query_string               = false
     },
+    # Behaviour to enable cookie forwarding for auth endpoints
+    local.is_auth ? [
+      {
+        path_pattern               = "/api/auth/*"
+        allowed_methods            = ["HEAD", "DELETE", "POST", "GET", "OPTIONS", "PUT", "PATCH"]
+        cache_policy_id            = local.managed_caching_disabled_policy_id
+        cached_methods             = ["GET", "HEAD"]
+        compress                   = true
+        origin_request_policy_id   = aws_cloudfront_origin_request_policy.front_end_auth.id
+        response_headers_policy_id = aws_cloudfront_response_headers_policy.front_end.id
+        target_origin_id           = "alb"
+        use_forwarded_values       = false
```

Comment:
As we are specifying a path_pattern for authentication, wouldn't we want to forward cookies? Should `use_forwarded_values` be set to true?